quicktest.go

```diff
@@ -33,34 +44,45 @@ func New(t testing.TB) *C {
 // uses the wrapped TB value to fail the test appropriately.
 type C struct {
 	testing.TB
-	cleanup func()
+	deferred func()
 }
 
-// AddCleanup registers a function to be called when c.Cleanup is
-// called. Cleanup functions will be called in last added, first called
+// Defer registers a function to be called when c.Done is
+// called. Deferred functions will be called in last added, first called
 // order.
-func (c *C) AddCleanup(f func()) {
-	oldCleanup := c.cleanup
-	c.cleanup = func() {
-		defer oldCleanup()
+func (c *C) Defer(f func()) {
+	oldDeferred := c.deferred
+	c.deferred = func() {
+		defer oldDeferred()
 		f()
 	}
 }
 
-// Cleanup calls all the functions registered by AddCleanup in reverse
-// order to the order they were registered. After it's called, the
-// cleanup functions are unregistered, so calling Cleanup twice will
-// only call the cleanup functions once.
+// Done calls all the functions registered by Defer in reverse
+// registration order. After it's called, the functions are
+// unregistered, so calling Done twice will only call them once.
 //
 // When a test function is called by Run, the C value passed into it
-// will be cleaned up automatically when it returns.
-func (c *C) Cleanup() {
-	// Note: we need to use defer in case the cleanup panics
+// Done will be called automatically when it returns.
```

Comment:
"... the C value passed into it Done will be called ..." Maybe rephrase this?

Reply:
done